### PR TITLE
Add landing page to the site

### PR DESCRIPTION
**Pull request by jshoe**
- Change routing to go to a new Landing.jsx component
- New landing page:
screen shot 2018-10-01 at 2 51 41 pm

- Other UI tweaks
- Removed sendMail link on the login page since we don't need it anymore (better to use the sign up form).
- Most of the diff in Login.jsx is not really new code

### CODE REVIEW

File: app/assets/src/components/IconComponent.jsx [hunk 1]
```diff
@@ -384,6 +386,223 @@ class IconComponent {
...
   }
+
+  static discover(color) {
```
Comment (tfrcarvalho) [inline app/assets/src/components/IconComponent.jsx]: What is this function?

Comment (tfrcarvalho) [reply app/assets/src/components/IconComponent.jsx]: We now have a folder for icons `ui/icons`. Define one new icon per file.

File: app/assets/src/components/Login.jsx [hunk 2]
```diff
@@ -102,92 +102,108 @@ class Login extends React.Component {
   }
 
   sendMail() {
-    var link =
-      "mailto:regger@chanzuckerberg.com?Subject=Request%20IDseq%20Access";
+    var link = "mailto:accounts@idseq.net?Subject=Request%20IDseq%20Access";
```
Comment (tfrcarvalho) [inline app/assets/src/components/Login.jsx]: Can we send the email from the server? 
If not, maybe obfuscate a bit with `<a href='mailt&#111;&#58;&#109;a&#105;lt%6F%3A&#97;ccou%6E&#116;s&#64;i&#100;&#115;eq&#46;net&#63;&#83;ub&#106;e%6&#51;%&#55;4=&#82;&#101;qu&#37;&#54;5st%&#50;5&#50;0ID&#37;73e%&#55;1%2&#37;3&#48;Acces&#115;'>&#109;&#97;&#105;&#108;to&#58;accoun&#116;s&#64;idseq&#46;net?Subject=Req&#117;es&#116;%2&#48;ID&#115;eq%20&#65;c&#99;&#101;&#115;s</a>`

File: app/assets/src/components/IconComponent.jsx (hunk 1 above)
Comment (jshoe) [reply app/assets/src/components/IconComponent.jsx]: It's the "discover" icon :O same as the others actually

Comment (jshoe) [reply app/assets/src/components/IconComponent.jsx]: like IconComponent.discover or IconComponent.mosquito

File: app/assets/src/components/Samples.jsx [hunk 3]
```diff
@@ -228,13 +228,7 @@ class Samples extends React.Component {
 
     $(document).ready(function() {
       $("select").material_select();
-      $(".modal").modal({
```
Comment (tfrcarvalho) [inline app/assets/src/components/Samples.jsx]: Unrelated changes? What was this?

File: app/assets/src/styles/ui/controls/buttons/transparent_button.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  min-width: 8rem;
+  margin-top: 0.4rem;
+
+  .icon-label {
+    svg {
+      fill: $white;
+    }
+  }
+
+  &:hover {
```
Comment (tfrcarvalho) [inline app/assets/src/styles/ui/controls/buttons/transparent_button.scss]: Maybe make the color white with a small opacity? Would it work?

Comment (jshoe) [reply app/assets/src/styles/ui/controls/buttons/transparent_button.scss]: you want the text color to be more transparent?

**Comment by jshoe** [comment]
going to remove schema.rb..

File: app/assets/src/styles/ui/controls/buttons/transparent_button.scss (hunk 4 above)
Comment (tfrcarvalho) [reply app/assets/src/styles/ui/controls/buttons/transparent_button.scss]: Just the background. And i do not think the blue work well for black background and definitely not for all transparent buttons.

**Comment by jshoe** [comment]
@tfrcarvalho cool, made all the requested changes I believe
